src/time_integration/methods_2N.jl

```diff
@@ -129,67 +128,84 @@ function solve(ode::ODEProblem, alg::T;
         error("unsupported")
     end
 
+    return integrator
+end
+
+# Fakes `solve`: https://diffeq.sciml.ai/v6.8/basics/overview/#Solving-the-Problems-1
+function solve(ode::ODEProblem, alg::T;
+               dt, callback = nothing, kwargs...) where {T <: SimpleAlgorithm2N}
```

Comment:
```suggestion
function solve(ode::ODEProblem, alg:: SimpleAlgorithm2N;
               dt, callback = nothing, kwargs...)
```

Same here. I think this is a bit easier to read